workflow/react-project-page/src/components/Main/MainTitle/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import classes from './MainTitle.module';
+
+const MainTitle = () => {
+  return (
+    <>
+      <h1 className={classes.mainTitle}>Cocktail App</h1>
+    </>
```

Comment:
you need to use React Fragment (i.e. `<></>`) when inside there's 2 or more elements, here there's one

Reply:
done